### PR TITLE
run provider tests in ocis

### PR DESCRIPTION
**Pull request by jasson99**
## Description:
- create a user with email address, as it is required in OCIS. 
- after recreating the user make an extra requests to have a workaround for  owncloud/ocis#1675
- on ocis skip the provider state that makes users subadmins of groups see https://github.com/owncloud/product/issues/289
- run the tests in drone and upload to pactflow

next step (in other PRs) will be to make as much test pass as possible and raise issues for the other ones

### CODE REVIEW

**Comment by individual-it** [comment]
issue about recreating the user: https://github.com/owncloud/ocis/issues/1675

File: .drone.star [hunk 1]
```diff
@@ -271,10 +368,45 @@ def consumerTestPipeline(subFolderPath = '/'):
...
+            'base': '/var/www/owncloud',
+            'path': 'owncloud-sdk'
+        },
+        'trigger': {
+            'branch': 'master'
+        },
+        'depends_on': [ 'testConsumer-root', 'testConsumer-subfolder' ],
+        'steps':
+            buildSystem() +
+            pactLog() +
```
Comment (dpakach) [inline .drone.star:389]: This step is useless since we moved to pactv3 but can be removed in different PR

**Review by dpakach — APPROVED** [review]
LGTM :+1:

**Review by Talank — COMMENTED** [review]
LGTM

**Review by dpakach — APPROVED** [review]

File: .drone.star (hunk 1 above)
Comment (individual-it) [reply .drone.star:394]: @dpakach PR in https://github.com/owncloud/owncloud-sdk/pull/741
